fighthealthinsurance/utils.py

```
@@ -120,12 +122,21 @@ def _normalize_role(role: str) -> str:
-        elif (result[0].get("role") == "system" and result[1] and result[1].get("role") == "assistant"):
-            logger.error("We should always start have a user message after system instead {result}")
+        elif (
+            result[0].get("role") == "system"
+            and len(result) > 1
+            and result[1]
+            and result[1].get("role") == "assistant"
+        ):
+            logger.error(
+                "We should always start have a user message after system instead {result}"
+            )
             result = [result[0]] + result[2:]
```

Ruff (0.14.10)

Consider `[result[0], *result[2:]]` instead of concatenation

Replace with `[result[0], *result[2:]]`

(RUF005)